packages/components/src/color-palette/index.native.js

```diff
@@ -175,6 +175,17 @@ function ColorPalette( {
 		}
 	}
 
+	function getColorGradientName( value ) {
+		const foundColorName = isGradientSegment
+			? defaultSettings.gradients.find(
+					( gradient ) => gradient.gradient === value
+			  )
+			: defaultSettings.allColors.find(
+					( color ) => color.color === value
+			  );
+		return foundColorName ? foundColorName?.name : '';
```

Comment:
What do you think of a fallback of the raw value when the color lacks a name? Alternatively, a label of "unlabeled color" or something similar. 
```suggestion
		return foundColorName ? foundColorName?.name : value;
```

Reply:
Good idea! What about: "Unlabeled color.  _hex value_"?

Reply:
Sounds good!